scholia/app/templates/base.html

```diff
@@ -108,6 +108,7 @@
 	 if ('en' in item.labels) {
 	     var title = item.labels.en.value
 	     $("#h1").text(title);
+	     $(".self").text(title);
```

Comment:
How is that related to the new panel?

Reply:
It addresses the issue of using the item's label in the section's explanatory text, as discussed [here](https://github.com/WDscholia/scholia/pull/2214#issuecomment-1367983965).